stable/mariadb/templates/master-statefulset.yaml

```diff
@@ -50,6 +50,18 @@ spec:
         - name: {{ . }}
       {{- end}}
       {{- end }}
+      {{- if .Values.master.persistence.enabled }}
+      initContainers:
+      - name: init-{{ template "mariadb.fullname" . }}
+        image: "{{ template "mariadb.image" . }}"
+        command: ["sh", "-c", "chmod 0777 /bitnami/mariadb"]
```

Comment:
Does this need the full `777`, or just be owned by the right user?

Reply:
Taking over from alex, I can't say.
Who would be the correct user ?

Reply:
Also note that https://github.com/SUSE/charts/pull/6 does the equivalent change for mongodb.

Reply:
Looking at the relevant [dockerfile](https://github.com/bitnami/bitnami-docker-mariadb/blob/96d18812ab8705b5197f4258a73ab8616b6839cf/10.1/debian-9/Dockerfile#L37), looks like it's `1001`?

Reply:
Yeah. Looks to be it. Will make the change. Currently not set up for testing it however. We will see.

Reply:
Change made with commit dd2449f18718ceb81f2dab88ab6fad329175bc11

Reply:
See also commit alexh/fix-mongodb-permissions over on https://github.com/SUSE/charts/pull/6